cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/TransformerConversionTest.java

```diff
@@ -565,7 +559,7 @@ private LoadFlowParameters defineLoadflowParameters(LoadFlowParameters loadFlowP
             case SPLIT:
                 t3wtSplitShuntAdmittance = true;
         }
-        boolean twtSplitShuntAdmittance = false;
+        boolean twtSplitShuntAdmittance;
```

Comment:
You can simplify it even more:
```suggestion
         if (t2wtSplitShuntAdmittance != t3wtSplitShuntAdmittance) {
            throw new PowsyblException(String.format("Unexpected SplitShuntAdmittance configuration %s %s",
                t2wtSplitShuntAdmittance, t3wtSplitShuntAdmittance));
        }
        boolean twtSplitShuntAdmittance = t2wtSplitShuntAdmittance;
```

Reply:
yes, thanks!